packages/wonder-blocks-data/src/hooks/__tests__/use-cached-effect.test.js

```diff
@@ -91,49 +100,94 @@ describe("#useCachedEffect", () => {
             },
         );
 
-        it("should not request data", () => {
-            // Arrange
-            const fakeHandler = jest.fn().mockResolvedValue("data");
+        it.each(Array.from(FetchPolicy.members()))(
+            "should not request data for FetchPolicy.%s",
```

Comment:
Could you add something to the title about the reason for it not requesting data?  It's because `fakeHandler` is resolved with data, right?

Reply:
No, it's because it's server-side. That's the entire context here. This hook should never invoke the effect when server-side. I feel like adding that to the test name duplicates information already held in the parent describe.